Loenn/entities/battery.lua

```diff
@@ -0,0 +1,10 @@
+local battery = {}
+
+battery.name = "FactoryHelper/Battery"
+battery.placements = {
+    name = "battery"
+}
+
+battery.texture = "objects/FactoryHelper/batteryBox/battery00"
```

Comment:
minor, but battery selection box is a bit large, might want to trim some in a selection function

Reply:
now returning a 8x14 rectangle